### ISSUE TITLE
Cloudformation Template for VPC Setup

### ISSUE THREAD

**Issue by marco-hoyer** [zalando-stups/sevenseconds#17]
Hi there,

I would prefer the resource creation to be done by cloudformation templates.  For VPC for example this would mean to ignore the default VPC and maybe always delete it. Would you be interested in a pull request implementing this? What was the intention of your implementation?

Greets

**Comment by apfeiffer85** [comment]
Hi, 

we already had cloudformation in our former tool (overlord) but we decide to use the API because its much faster and it is easier to change something. If we want to change something with cloudformation it is mostly necessary to rollback the whole stack, thats not a option for the initial setup. 

So we are really happy with this "API Solution" and it is pretty fast to make the initial setup. 

But we also use Cloudformation, for example in our deploy tool "senza" use cloudformation.

**Comment by hjacobs** [comment]
@marco-hoyer yes, I think going for a "low-tech" approach ("low-tech" is my favorite euphemism for "hacked together") instead of Cloud Formation (CF) is appropriate for the basic account setup as we have more control for migrations (updates of VPC/account config) --- immutability in general is good, but if a VPC change requires us to wait for the team to migrate servers (shutdown, start in new VPC etc) it just slows us (STUPS team) down. I don't feel comfortable about letting AWS CF decide when to rollback and when to update.
